src/lib/file-system/file-watcher.ts

```diff
@@ -38,8 +41,11 @@ export function createFileWatch(
     });
   };
 
-  return Observable.create((observer) => {
-    watch.on('all', (event: AllFileWatchEvents, filePath: string) => handleFileChange(event, filePath, observer));
-    return () => watch.close();
-  });
+  return {
+    watcher: watch,
+    onFileChange: Observable.create((observer) => {
```

Comment:
```suggestion
    onFileChange: new Observable((observer) => {
```
The other is deprecated 😄  ❤️

Reply:
Haha, at the moment the Linux failure is more serious than this deprecation.